### PR TITLE
Use offset instead of tzdb id

### PR DESCRIPTION
**Pull request by Hosshii**
**What this PR does / why we need it**:
Using tzdb ID may requires tzdb. It cause problem when tzDB does not esits (such as alpine linux).
So use offset instead of tzdb ID.

**Which issue(s) this PR fixes**:

Fixes #3465

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/server/service/webservice/service.proto [hunk 1]
```diff
@@ -438,8 +438,8 @@ message GetInsightDataRequest {
     int64 range_from = 2 [(validate.rules).int64.gt = 0];
     int64 range_to = 3 [(validate.rules).int64.gt = 0];
     string application_id = 4;
-    // tzdb Id like `Asia/Tokyo`. null means UTC.
-    string timezone = 5;
+    // seconds east of UTC
+    int32 offset = 5;
```
Comment (khanhtc1202) [inline pkg/app/server/service/webservice/service.proto:442]: should be int64 🤔

Comment (Hosshii) [reply pkg/app/server/service/webservice/service.proto:442]: A day is 86400 seconds, so I think int32 is sufficient (go [uses](https://pkg.go.dev/time#FixedZone) int)

**Comment by knanao** [comment]
NIce
/approve

**Pull request merged**